modules/plugin/jdbc/jdbc-sqlserver/src/main/java/org/geotools/data/sqlserver/reader/SqlServerBinaryReader.java

```diff
@@ -75,7 +76,12 @@ public Geometry read(byte[] bytes) throws IOException {
     }
 
     public Geometry read(InStream is) throws IOException {
-        parse(is);
+        try {
+            parse(is);
+        } catch (ParseException e) {
+            e.printStackTrace();
```

Comment:
I do not enjoy printStackTrace()

Reply:
I was wondering why PMD did not catch this... and turns out, the rule is not configured. But it's available:
https://pmd.github.io/pmd-6.32.0/pmd_rules_java_bestpractices.html#avoidprintstacktrace

Must add this one to my TODO list (but even happier if someone wants to beat me to it)